docs/products/kafka/howto/manage-acls.md

```diff
@@ -1,77 +1,452 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
 
-For detailed information on permission mapping,
-see [Access control lists and permission mapping](/docs/products/kafka/concepts/acl).
+Access control lists (ACLs) in Aiven for Apache Kafka® define permissions for topics, schemas, consumer groups, and transactional IDs.
+ACLs control which authenticated users or applications (principals) can perform specific
+operations on these resources.
 
-To manage users and ACL entries, access the corresponding sections in
-the left sidebar of the service page within the [Aiven
-Console](https://console.aiven.io/).
+## Types of ACLs
 
-## Add a user
+Aiven for Apache Kafka® supports two types of ACLs:
 
-:::warning
-A user with the `Admin` permission can create topics with any name, as
-the `CreateTopics` permission is applied at the cluster level.
+- **Aiven ACLs**: These provide topic-level permissions and support wildcard patterns.
+- **Kafka-native ACLs**: These offer advanced, resource-level permissions with `ALLOW`
+  and `DENY` rules for operations on multiple resource types, including topics, groups,
+  and clusters.
 
-All other permissions related to a topic (`Alter`, `Delete`) **only**
-apply to the topics matching the pattern that you specify.
+:::note
+ACL restrictions currently do not apply to Kafka REST. Rules are applied based on the
+username and topic names, but there are no restrictions on consumer group names.
+
+Support for extending these restrictions to Kafka REST is under development.
 :::
 
-To add a new user:
+## Add a Kafka-native ACL entry
+
+<Tabs groupId="acl-methods">
+<TabItem value="console" label="Aiven Console" default>
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and select your
+   service.
+1. Click <ConsoleLabel name="acl" />
+1. Click **Add entry**.
+1. On the **Add access control entry** screen:
+   1. Select **Kafka-native ACLs** as the ACL type.
+   1. Fill in the following fields:
+      1. **Permission type**: Select `ALLOW` or `DENY`.
+      1. **Principal**:Enter the principal in the format `User:<username>`.
+      1. **Operation**: Select the operation, such as `Read` or `Write`.
+      1. **Resource type**: Select the Apache Kafka resource to manage.
+      1. **Pattern type**: Select `LITERAL` for exact matches or `PREFIXED` for
+         pattern-based matches.
+      1. **Resource**: Enter the resource name or a prefix for pattern-based matching.
+      1. **Host**: Enter the allowed host, or use `*` for all.
+   1. Click **Submit**.
+
+</TabItem>
+<TabItem value="cli" label="Aiven CLI">
+
+To add an Kafka-native ACL entry with the Aiven CLI, run:
+
+```bash
+avn service kafka-acl-add <service_name> \
+  --principal <principal> \
+  --operation <operation_type> \
+  --resource-type <resource_type> \
```

Comment:
There is no `--resource-type` in final version. I think it was in some interim plan to have it such. Final version has one of `--topic <topic> | --cluster | --group <group> | --transactional-id <transaction-id>`

Reply:
Please take a look at the updated code sample.